component/core/src/main/java/org/exoplatform/social/core/manager/IdentityManagerImpl.java

```diff
@@ -216,7 +228,6 @@ public InputStream getBannerInputStream(Identity identity) throws IOException {
     }
     return identityStorage.getBannerInputStreamById(identity);
   }
-  
```

Comment:
Unecessary change

Reply:
i undo the change, it's just bad detected in github 
![Screenshot from 2022-11-16 14-35-10](https://user-images.githubusercontent.com/70575401/202194667-8bd23f9d-f669-4559-9ce5-801ecf06ff3d.png)